tensorflow/stream_executor/blas.h

```diff
@@ -1383,6 +1455,93 @@ class BlasSupport {
                           const DeviceMemory<std::complex<double>> &a, int lda,
                           DeviceMemory<std::complex<double>> *b, int ldb) = 0;
 
+  // Creates a backend-specific plan object for a blaslt matmul operation, which
+  // can then be passed to DoBlasLtMatmul(). When possible, plans should be
+  // created once and reused for multiple calls to DoBlasLtMatmul().
+  // Returns a null pointer on failure.
+  std::unique_ptr<blas::IBlasLtMatmulPlan> CreateBlasLtMatmulPlan(
```

Comment:
Can we create a `struct MatmulPlanParams` for it? Something like:
```
struct MatmulPlanParams {
  blas::DataType ab_type;
  blas::DataType c_type;
  ...
  int64 lda;
  int64 stride_a = 0;
  int64 ldb;
  int64 stride_b = 0;
  int64 ldc;
  int64 stride_c = 0;
};
```

This way the function signature is simpler and can be updated all at once through extending MatmulPlanParams.

Reply:
Done